src/assets/AccountTrackerController.ts

```diff
@@ -10,9 +10,11 @@ import { BNToHex, query, safelyExecuteWithTimeout } from '../util';
  * Account information object
  *
  * @property balance - Hex string of an account balancec in wei
+ * @property importTime - Data time when an account as created/imported
```

Comment:
Hmm - I'm not sure this is the best place for this data :thinking: 

This controller is effectively a cache of account balances right now. It was based upon the `account-tracker.js` controller in the extension, which still has mostly the same functionality today. The state of this controller is purely _transient_ - we don't persist any of it.

This `importTime` value is different in that it's totally unrelated to balance or blockchain state in general. And since it's not a cache, we'd need to persist it.

Do you think we could move this to the `PreferencesController` instead, to be a part of the `identity` object? That state is similar in that it's entire local (not based upon the blockchain), and must be persisted. It should include a complete account of all wallet accounts as well (accounts get stored there before they are synced with this controller).

Reply:
I created an issue recently how to improve our identity management: https://github.com/MetaMask/controllers/issues/431

Our identity state is spread all over the place now unfortunately. At some point it would be great to have an "identity conttroller" that would consolidate most of it. That would seem like the idea place for this `importTime` as well.

Reply:
I was debating between AccountTracker & Preferences Controllers. I will make that change.